Option to disable systemd unit installation

Otherwise this error happens in Alpine Linux. I'll patch it out for now in  the packaging.
```
--  is the systemd user unit file install dir
CMake Error at data/CMakeLists.txt:28 (install):
  install FILES given no DESTINATION!
```

--- a/data/CMakeLists.txt
+++ b/data/CMakeLists.txt
@@ -21,21 +21,24 @@ add_schema (${SCHEMA_NAME})
 ##  Systemd Unit File
 ##
 
-# where to install
-pkg_get_variable(SYSTEMD_USER_DIR systemd systemduserunitdir)
-message (STATUS "${SYSTEMD_USER_DIR} is the systemd user unit file install dir")
+pkg_check_modules(SYSTEMD systemd)
+if (${SYSTEMD_FOUND})
+    # where to install
+    pkg_get_variable(SYSTEMD_USER_DIR systemd systemduserunitdir)
+    message (STATUS "${SYSTEMD_USER_DIR} is the systemd user unit file install dir")
 
-set (SYSTEMD_USER_NAME "${CMAKE_PROJECT_NAME}.service")
-set (SYSTEMD_USER_FILE "${CMAKE_CURRENT_BINARY_DIR}/${SYSTEMD_USER_NAME}")
-set (SYSTEMD_USER_FILE_IN "${CMAKE_CURRENT_SOURCE_DIR}/${SYSTEMD_USER_NAME}.in")
+    set (SYSTEMD_USER_NAME "${CMAKE_PROJECT_NAME}.service")
+    set (SYSTEMD_USER_FILE "${CMAKE_CURRENT_BINARY_DIR}/${SYSTEMD_USER_NAME}")
+    set (SYSTEMD_USER_FILE_IN "${CMAKE_CURRENT_SOURCE_DIR}/${SYSTEMD_USER_NAME}.in")
 
-# build it
-set (pkglibexecdir "${CMAKE_INSTALL_FULL_PKGLIBEXECDIR}")
-configure_file ("${SYSTEMD_USER_FILE_IN}" "${SYSTEMD_USER_FILE}")
+    # build it
+    set (pkglibexecdir "${CMAKE_INSTALL_FULL_PKGLIBEXECDIR}")
+    configure_file ("${SYSTEMD_USER_FILE_IN}" "${SYSTEMD_USER_FILE}")
 
-# install it
-install (FILES "${SYSTEMD_USER_FILE}"
-         DESTINATION "${SYSTEMD_USER_DIR}")
+    # install it
+    install (FILES "${SYSTEMD_USER_FILE}"
+             DESTINATION "${SYSTEMD_USER_DIR}")
+endif()
 
 ##
 ##  XDG Autostart Config File

